x/gov/client/cli/prompt.go

```diff
@@ -135,15 +135,17 @@ type proposalType struct {
 
 // Prompt the proposal type values and return the proposal and its metadata
 func (p *proposalType) Prompt(cdc codec.Codec) (*proposal, types.ProposalMetadata, error) {
-	proposal := &proposal{}
-
 	// set metadata
 	metadata, err := Prompt(types.ProposalMetadata{}, "proposal")
 	if err != nil {
 		return nil, metadata, fmt.Errorf("failed to set proposal metadata: %w", err)
 	}
-	// the metadata must be saved on IPFS, set placeholder
-	proposal.Metadata = "ipfs://CID"
+
+	proposal := &proposal{
+		Metadata: "ipfs://CID",
```

Comment:
```suggestion
		Metadata: "ipfs://CID", // the metadata must be saved on IPFS, set placeholder
```

I think the comment still has a value.